static/src/scripts/app/mixens/BaseModelMixen.coffee

```diff
@@ -1,21 +1,12 @@
 define [
-  'jquery'
-  'underscore'
   'backbone'
   'mixen'
   'mixens/AuthMixen'
-  'mixens/LoadedMixen'
-], ($, _, backbone, Mixen, AuthMixen, LoadedMixen) ->
-  class BaseModel extends Mixen(LoadedMixen, AuthMixen, Backbone.Model)
+  'mixens/SyncingMixen'
+], (backbone, Mixen, AuthMixen, SyncingMixen) ->
+  class BaseModel extends Mixen(SyncingMixen, AuthMixen, Backbone.Model)
     urlRoot: jailbreak.api_host
 
-    getRenderContext: ->
-      context = super ? {}
-
-      data =
-        error: @error
-        errorMessage: @errorMessage
-        errorStatus: @errorStatus
-        loaded: @loaded
-
-      _.extend context, data
+    sync: ->
+      # don't remove
```

Comment:
Again another great comment. I should provide a reason. It's about the Mixens `SyncingMixen` and `AuthMixen` implementing `sync`. We don't want to break the chain.